src/components/OrdersWidget/index.tsx

```diff
@@ -101,12 +103,15 @@ const OrdersWidget: React.FC = () => {
   const [filteredOrders, setFilteredOrders] = useSafeState<FilteredOrdersState>(emptyState())
   const [selectedTab, setSelectedTab] = useSafeState<OrderTabs>('active')
 
+  // Subscribe to trade events
+  const trades = useTrades()
+
   // syntactic sugar
   const { displayedOrders, displayedPendingOrders, markedForDeletion } = useMemo(
     () => ({
-      displayedOrders: filteredOrders[selectedTab].orders,
-      displayedPendingOrders: filteredOrders[selectedTab].pendingOrders,
-      markedForDeletion: filteredOrders[selectedTab].markedForDeletion,
+      displayedOrders: selectedTab === 'fills' ? [] : filteredOrders[selectedTab].orders,
+      displayedPendingOrders: selectedTab === 'fills' ? [] : filteredOrders[selectedTab].pendingOrders,
+      markedForDeletion: selectedTab === 'fills' ? new Set<string>() : filteredOrders[selectedTab].markedForDeletion,
```

Comment:
I wonder if it could be a good idea to have `emptyArray, emptySet` here fr stable references, like @W3stside did somewhere else.
Save a couple useMemo calls
But ugly